main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/PuerkitoBio/goquery"
-	"github.com/daviddengcn/go-colortext"
+	ct "github.com/daviddengcn/go-colortext"
```

Comment:
Thank your contribution. Please revert this.

Reply:
I revert it.